packages/engine/src/scene/systems/SceneObjectDynamicLoadSystem.ts

```diff
@@ -63,7 +82,34 @@ export default async function SceneObjectDynamicLoadSystem(world: World) {
             position: data.position
           })
 
+          hasComponent(entity, RigidBodyComponent) && Physics.removeCollidersFromRigidBody(entity, world.physicsWorld)
+          const targetNode = nodeMap.get(entity)
+          if (targetNode) {
+            iterateEntityNode(targetNode, (node) => {
```

Comment:
what is the case in which a child entity is separated in distance to it's parent? the dynamic load should only ever be used in leaf nodes of the entity tree, or alternatively parent nodes should with dynamic loading should stop their children from being loaded, and then handle the loading/unloading of their children

Reply:
Any child that has a position offset will be separated in distance from its parent. Used when wanting to dynamically load an object or group of objects as the player approaches a position in the level.